enlighten/Controller.py

```diff
@@ -1848,6 +1848,8 @@ def process_reading(self, reading, spec=None, dark=None, ref=None, settings=None
         ########################################################################
         # Graph Post-Processed Spectrum
         ########################################################################
+        if self.form.ui.checkBox_despike_enable.isChecked():
+            pr = self.despiking_feature.process(pr)
```

Comment:
You need to talk to Dieter about where he would insert this in the order of operations. For my part, I would prefer this be BEFORE boxcar and plugins.

Reply:
He might actually want it before we snap the "recordable_dark"